src/pages/Meeting/DetailMeatingManagePage.tsx

```
@@ -125,17 +157,21 @@ export default function DetailMeetingManagePage() {
+                  {getMembersInGroup(groupName).map((member : MeetingMemberItem) => (
                     <div
-                      className="bg-[#F4F2F1] w-[365px] h-[44px] rounded-[20px] flex items-center px-3 text-[17px] font-medium text-[#2C2C2C] truncate"
+                      className="bg-[#F4F2F1] w-[365px] h-[44px] rounded-[20px] flex items-center px-3 text-[17px] font-medium text-[#2C2C2C] truncate cursor-pointer hover:shadow-sm"
+                      onClick={() => {
+                        removeMemberFromGroup(groupName, member.memberInfo.nickname);
+                      }}
                     >
-                      <img src={member.basicInfo.profileImageUrl} className="w-[32px] h-[32px] rounded-full bg-[#DADADA] mr-5 text-[var(--Gray-1,#2C2C2C)] font-pretendard text-[18px] font-medium leading-[135%]" />
-                      {member.basicInfo.nickname}
+                      <img src={member.memberInfo.profileImageUrl || "/assets/basic_profile.png"}
+                      className="w-[32px] h-[32px] rounded-full mr-5 text-[var(--Gray-1,#2C2C2C)] font-pretendard text-[18px] font-medium leading-[135%]" />
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)